### PR TITLE
Added sample code for BottomNavigationBar widget [#21136]

### PR DESCRIPTION
**Pull request by NPKompleet**
Added a sample code to the BottomNavigationBar documentation

### CODE REVIEW

File: packages/flutter/lib/src/material/bottom_navigation_bar.dart [hunk 1]
```diff
@@ -67,6 +67,63 @@ enum BottomNavigationBarType {
...
+///      body: Center(
+///        child: _widgetOptions.elementAt(_selectedIndex),
+///      ),
+///      bottomNavigationBar: BottomNavigationBar(
+///        items: <BottomNavigationBarItem>[
+///          BottomNavigationBarItem(icon: Icon(Icons.home), title: Text('Home')),
+///          BottomNavigationBarItem(
+///              icon: Icon(Icons.business), title: Text('Business')),
+///          BottomNavigationBarItem(
+///              icon: Icon(Icons.school), title: Text('School')),
```
Comment (Hixie) [inline packages/flutter/lib/src/material/bottom_navigation_bar.dart]: nit: let's have all three BottomNavigationBarItem either be wrapped fully or all be on one line... it looks a bit weird with them differently wrapped.

**Comment by Hixie** [comment]
This is great! Thanks for your contribution! Please let me know if you're able to make the suggested very trivial change and then I'll eagerly land your PR!

**Comment by NPKompleet** [comment]
@Hixie I just added the fixes you recommended to the BottomNavigationBar sample code.

**Comment by Hixie** [comment]
Thanks for your contribution!

**Pull request merged**